learning-basics/configure-yaml/src/main/resources/hazelcast-client.yaml

```diff
@@ -0,0 +1,6 @@
+hazelcast-client:
+  import:
```

Comment:
I guess this import is for the reader to observe something, would it make sense to describe it in README?

Reply:
Added to the README that the first test demonstrates composing YAML configuration files.